### PR TITLE
[REVIEW][FEATURE][GAWE-64] - Implement CreateNewPassword UI

### PR DESCRIPTION
**Pull request by fauzanramadhani**
#### Description

Implement CreateNewPassword UI

#### Ticket Reference(s)

[GAWE-64](https://bluehabit.atlassian.net/browse/GAWE-64)

##### UI Changes

image

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

@triandamai @WhyUReRenldI @azkifairuz 

[GAWE-64]: https://bluehabit.atlassian.net/browse/GAWE-64?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ *
+ * Unauthorized copying, publishing of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+
+package com.bluehabit.eureka.feature.authentication.resetPassword.screen
+
+import androidx.lifecycle.ViewModel
+
+class ResetPasswordViewModel : ViewModel() {
```
Comment (triandamai) [inline feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt:12]: harus extends `MviViewModel`

File: feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright © 2023 Blue Habit.
+ *
+ * Unauthorized copying, publishing of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+
+package com.bluehabit.eureka.feature.authentication.resetPassword.screen
```
Comment (triandamai) [inline feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt:8]: kenapa ada di dalem screen?

**Review by triandamai — CHANGES_REQUESTED** [review]
cek

File: feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt (hunk 2 above)
Comment (fauzanramadhani) [reply feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt]: done

File: feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt (hunk 1 above)
Comment (fauzanramadhani) [reply feature/feature-authentication/src/main/java/com/bluehabit/eureka/feature/authentication/resetPassword/screen/ResetPasswordViewModel.kt]: done

**Review by triandamai — APPROVED** [review]
LGTM!

**Pull request merged**